### PR TITLE
add IE friendly checks for blur and focus to avoid script breaking errors

### PR DESCRIPTION
**Pull request by mschulera**
* in order to remove certain error messages in IE the conditions checking for element blur were edited and condition checking for element to focus was added

### CODE REVIEW

File: src/featherlight.js [hunk 1]
```diff
@@ -619,7 +619,7 @@
 
 				this._$previouslyWithTabIndex.add(this._$previouslyTabbable).attr('tabindex', -1);
 
-				if (document.activeElement.blur) {
+				if (document.activeElement) {
```
Comment (marcandre) [inline src/featherlight.js:622]: I can't easily check this, but #296 reports that some elements don't support the `blur` method? Maybe best to check both `document.activeElement && document.activeElement.blur`?

File: src/featherlight.js [hunk 2]
```diff
@@ -525,7 +525,7 @@
 				} else if(fl.persist !== false) {
 					$target.data('featherlight-persisted', fl);
 				}
-				if (elemConfig.$currentTarget.blur) {
+				if (elemConfig.$currentTarget) {
```
Comment (marcandre) [inline src/featherlight.js:528]: `$currentTarget` should be a jQuery set, so the previous test is silly. I'm not sure your replacement is needed, but can't hurt.

Comment (mschulera) [reply src/featherlight.js:528]: @marcandre it is probably an edge case, but IE11 cannot run the if condition if elemConfig.$currentTarget is NULL and this prohibits featherlight from closing the modal correctly.

File: src/featherlight.js (hunk 1 above)
Comment (mschulera) [reply src/featherlight.js:622]: @marcandre I'd have to try this on the corresponding project in IE11, but I think this also will result in a script breaking error when the element targeted is not available/null

File: src/featherlight.js (hunk 2 above)
Comment (marcandre) [reply src/featherlight.js:528]: I understand, but in what circumstances would `$currentTarget` be `null`?

File: src/featherlight.js (hunk 1 above)
Comment (marcandre) [reply src/featherlight.js:622]: How would adding a check `&& document.activeElement.blur` break?

File: src/featherlight.js (hunk 2 above)
Comment (mschulera) [reply src/featherlight.js:528]: I have to re-check if the issue still persists, however it was the case in IE11 on some specific elements. We had created this workaround for a customer specific project.
